llvm/lib/Target/AMDGPU/VOP2Instructions.td

```diff
@@ -319,7 +319,7 @@ multiclass
 multiclass
     VOP2eInst<string opName, VOPProfile P, SDPatternOperator node = null_frag,
               string revOp = opName, bit useSGPRInput = !eq(P.NumSrcArgs, 3)>
-    : VOP2eInst_Base<opName, P, -1, "", node, revOp, useSGPRInput>;
+    : VOP2eInst_Base<opName, P, -1, "v_cndmask_b16", node, revOp, useSGPRInput>;
```

Comment:
We can pass vOPDName explicitly here in order to give this "fake" record more information, `defm V_CNDMASK_B16 : VOP2eInst <"v_cndmask_b16", VOP2e_I16_I16_I16_I1_fake16>;`. I do not see it as necessary though. The opcode is used in one location and it would not have been able to query this information before this change anyway. `VOPDOp` is given a value of `-1`, indicating a fake record as well.

Reply:
I don't follow why this is hardcoding v_cndmask_b16

Reply:
Otherwise the empty string will segfault on `!substr`. We can do this or check the indices, and the class is only used in one location.

Reply:
substr segfaults on an empty string!? That's a really bad tablegen bug

Reply:
Yes, replaced it with `v_` instead. Thanks.

Reply:
> substr segfaults on an empty string!? That's a really bad tablegen bug.

@jofrn if you have time can fix this too please (in a separate patch).

Reply:
Thanks. I merged at this point and fixed the `-1` -> `0` switch you made at the end.